.github/workflows/build-release.yml

```diff
@@ -3,6 +3,8 @@ name: "[builder] CI for releases"
 on:
   repository_dispatch:
     types: build
+  watch:
+    types: [started]
```

Comment:
Unnecessary changes

Reply:
This is a very useful method to debug Docker.